minerl/env/core.py

```diff
@@ -47,6 +48,8 @@
 
 missions_dir = os.path.join(os.path.dirname(__file__), 'missions')
 
+launch_instance_lock = FileLock(os.path.join(os.path.dirname(__file__), 'env.lock'))
```

Comment:
FileLock uses a file as communication for processes to proceed in the code. When a process enters the launch instance code, FileLock creates a file `env.lock` in the same directory and prevents other processes from entering unless the lock is removed.